Pipelines/test-execution-pipeline.yml

```diff
@@ -0,0 +1,102 @@
+parameters:
+  - name: IsMultiRegionEnabled
+    type: boolean
+    default: false
+
+  - name: IsClusterRequired
+    type: boolean
+    default: false
+
+  - name: JmeterFolderPath
```

Comment:
rename JmeterFolderPath to JMeterFolderPath

Reply:
updated the names